src/ui/coordinates/coordView.ts

```diff
@@ -0,0 +1,103 @@
+import h from 'mithril/hyperscript'
+import Chessground from '~/chessground/Chessground'
+import redraw from '~/utils/redraw'
+import { header } from '../shared/common'
+import Board from '../shared/Board'
+import i18n from '../../i18n'
+import layout from '../layout'
+import { INITIAL_FEN } from 'chessops/fen'
+import CoordCtrl from './coordCtrl'
+
+let orientation = (Math.random() > 0.5 ? 'white' : 'black') as Color
+let random = true
+
+function reOriente(color: Color | undefined): void {
+  random = false
+  if (color === undefined) {
+    color = Math.random() > 0.5 ? 'white' : ('black' as Color)
+    random = true
+  }
+
+  orientation = color
+  redraw()
+}
+
+export default function view(ctrl: CoordCtrl) {
+  const board = h(Board, {
+    key: orientation,
+    variant: 'standard',
+    chessground: new Chessground({
+      fen: INITIAL_FEN,
+      orientation: orientation,
+      coordinates: false,
+      movable: {
+        free: false,
+        color: null,
+      },
+      events: {
+        select: ctrl.handleSelect.bind(ctrl),
+      },
+    }),
+  })
+
+  const isWrongAnswer = (index: number) =>
+    ctrl.wrongAnswer === true && index === 0 ? '.nope' : ''
+
+  return layout.board(header(i18n('coordinateTraining')), [
+    h('main#trainer.coord-trainer.training', [
+      h('div.coord-trainer__side', { style: { visibility: ctrl.started } }, [
+        h('div.box', [h('h1', i18n('coordinates'))]),
+        h('form.color.buttons', [
+          h('group.radio', [
+            h('div', [
+              h('input#coord_color_3', {
+                type: 'radio',
+                name: 'color',
+                value: 3,
+                onclick: () => reOriente('black'),
```

Comment:
We shouldn't use `onClick` at all in the app to be consistent with the other buttons. Instead use `ontap` handler, from `~/ui/helper/index.ts`.
Ex usage here: https://github.com/veloce/lichobile/blob/master/src/ui/home/homeView.tsx#L211

Reply:
`ontap` works only with buttons these are inputs radio, the problem is when I changed them to buttons when I tap them they navigate out the home page I couldn't debug the problem